parlai/chat_service/core/chat_service_manager.py

```diff
@@ -0,0 +1,677 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import threading
+import time
+import traceback
+from parlai.chat_service.core.agents import ChatServiceAgent
+import parlai.chat_service.services.messenger.server_utils as server_utils
+import parlai.chat_service.services.messenger.shared_utils as shared_utils
+from parlai.chat_service.services.messenger.world_runner import MessengerWorldRunner
+from abc import ABC, abstractmethod
+
+
+parent_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
is this still needed?

Reply:
Isn't being used anywhere so guess not, removing it!